libgc/os_dep.c

```diff
@@ -32,7 +32,7 @@
       /* prototypes, so we have to include the top-level sigcontext.h to    */
       /* make sure the former gets defined to be the latter if appropriate. */
 #     include <features.h>
-#     if 2 <= __GLIBC__
+#     if 2 <= __GLIBC__ || !defined(__GLIBC__)
```

Comment:
Yes, I know. While here when `__GLIBC__` is not defined it mimics the behaviour of a modern libc, in exceptions-arm.c instead I had the problem that sigcontext was defined elsewhere, so removing the include fixed the problem.